### PR TITLE
PRO-425 Gas Fee Payment Asset Selection

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
* Select Gas fee asset

https://user-images.githubusercontent.com/78720269/190658768-0d6c21cc-b5a4-4c47-9058-13797ccd930e.mp4

### CODE REVIEW

File: src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    onSelectAsset(assets[0]);
+  }, [chain]);
+
+  const renderItem = (item, index) => {
+    const { iconUrl, symbol } = item;
+    return (
+      <Container
+        style={containerStyle}
+        key={'asset__' + index.toString()}
+        onPress={async () => {
```
Comment (IAmKio) [inline src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx:57]: Do we need async here?

File: src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          onSelectAsset(item);
+        }}
+      >
+        <ItemContainer isSelected={selectAsset?.symbol === symbol}>
+          <RowContainer>
+            <TokenIcon size={24} url={iconUrl} />
+            <Title style={left10}>
+              {item?.balance
+                ? item.balance.balance?.toFixed(1) + ' ' + symbol + '  •  ' + item?.formattedBalanceInFiat
```
Comment (IAmKio) [inline src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx:66]: Consider using template literals in future

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor points but other than that - great work!

File: src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx]: Resolved it!

File: src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx (hunk 2 above)
Comment (hemadri-tank-bacancy) [reply src/screens/Bridge/Exchange-CrossChain/GasFeeAssetSelection.tsx]: Resolved it!

**Review by IAmKio — APPROVED** [review]
LGTM and thank you

**Pull request merged**